readme.md

```diff
@@ -126,6 +134,8 @@ It exposes an additional `.all` stream, with `stdout` and `stderr` interleaved.
 
 The spawned process can be canceled with the `.cancel()` method on the promise, which causes the promise to reject an error with a `.isCanceled = true` property, provided the process gets canceled. The process will not be canceled if it has already exited.
 
+The spawned process can be killed with `.kill([signal], [options])` which allows you to retry the process killing at a later time (default: after 5 seconds)
```

Comment:
Maybe this should be rephrased to emphasize:
  - the retry is automatically done
  - it does not "try" to kill, it forces it

Reply:
Fixed: 2fb0ae0a7664c4c240d3ae9f764b6b0976549827